auth_app/templates/auth_app/home.html

```diff
@@ -0,0 +1,106 @@
+                    <a class="collapse-item">
+                        EDA Games is the proposal devised by the Eventbrite engineering team for the selection process of developers with junior profiles.
+                        We are convinced that playing is one of the best ways to learn, and also to demonstrate your knowledge and skills while having fun enjoying the process.
+                    </a>
+                </div>
+            </div>
+            <!-- Question 3 -->
+            <a class="card-header" href="#" data-toggle="collapse" data-target="#answerThree"
+                aria-expanded="true" aria-controls="answerThree">
+                <span>So, how you can play?</span>
+            </a>
+            <div id="answerThree" class="collapse" aria-labelledby="headingThree" data-parent="#FaqBarHome">
+                <div class="bg-white py-2 collapse-inner rounded ml-3">
+                    <a class="collapse-item">
+                        To participate it is necessary to have a bot created. 
+                        You will make a code for your bot to play against the bots of the other participants. 
+                        If you go to 
+                        <a href="{% url 'development:mybots' %}"> 
+                            My Bots 
+                        </a>
+                        menu, you will see that you already have one created. 
+                        This is your official bot whose name is your email username. 
+                        Once you have your code written (which you can always improve), 
+                        you can create challenges for other participants from 
+                        <a href="{% url 'development:challenge' %}"> 
+                            Challenge 
+                        </a> 
+                        menu and then check your results in 
+                        <a href="{% url 'development:match_history' %}"> 
+                            My Matches  
+                        </a> 
+                        menu.
+                    </a>
+                </div>
+            </div>
+            <!-- Question 4 -->
+            <a class="card-header" href="#" data-toggle="collapse" data-target="#answerFour"
+                aria-expanded="true" aria-controls="answerFour">
+                <span>About challenges and tournaments</span>
+            </a>
+            <div id="answerFour" class="collapse" aria-labelledby="headingFour" data-parent="#FaqBarHome">
+                <div class="bg-white py-2 collapse-inner rounded ml-3">
+                    <a class="collapse-item">
+                        As the end user of the platform, challenge other players' bots by creating matches. 
+                        Also from the Eventbrite team we can create tournaments where we will select a list of 
+                        bots to create 2-player games. You can check the results of 
+                        Tournament Results.
```

Comment:
I think that a better understanding of challenges could be.

```
You can prove your code performance by challenging other users in normal matches or participating in a tournament.
Only the EDA staff (admins) can create tournaments, but any player can participate in them.
In the final tournament, every single player will have to play with his <b>official bot</b> only.
```

Reply:
ok, I'm going to add this in the next PR.